modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,69 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by Organization Administrators, developers, and end-users; each of these roles has different privileges when connecting applications,
+and there are some restrictions to ensure sensitive data remains secure.
+
+== Organization Administrator
+
+An Organization Administrator is the only role that can view and manage Connected Apps in Access Management.
```

Comment:
I'm not sure about capping Connected Apps. I would lc all, but Kim might have diff advice.

Reply:
```suggestion
An Organization Administrator is the only role that can view and manage connected apps in Access Management.
```

Reply:
I don't think it should be capped unless I'm referring to the page at this point.